### PR TITLE
Use absolute imports in the template

### LINKED ISSUE #6: Add configuration for mypy
By default pre-commit mypy uses the following:

```ini
[mypy]
ignore_missing_imports = true
follow_imports = silent
show_column_numbers = true
```

Also consider adding `disable_error_code = misc` to avoid `error: Relative import climbs too many namespaces`.

### LINKED ISSUE #11: Use absolute imports and move test directory to root

### PR DESCRIPTION
**Pull request by Joonalai**
Fixes #11, closes #6.

### CODE REVIEW

File: {{cookiecutter.project_directory}}/test/test_plugin.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from {{cookiecutter.plugin_package}}.qgis_plugin_tools.tools.resources import plugin_name
+
+
+def test_plugin_name():
+    assert plugin_name() == "{{cookiecutter.plugin_name|replace(' ', '')}}"
```
Comment (LKajan) [inline {{cookiecutter.project_directory}}/test/test_plugin.py:5]: What is the plugin name the plugin_name() returns? Why it doesn't include spaces?

File: {{cookiecutter.project_directory}}/{{cookiecutter.plugin_package}}/plugin.py [hunk 2]
```diff
@@ -5,9 +5,13 @@
 from PyQt5.QtWidgets import QAction, QWidget
 from qgis.gui import QgisInterface
 
-from .qgis_plugin_tools.tools.custom_logging import setup_logger, teardown_logger
-from .qgis_plugin_tools.tools.i18n import setup_translation, tr
-from .qgis_plugin_tools.tools.resources import plugin_name
+{% if cookiecutter.plugin_package|length < 7 %}from {{cookiecutter.plugin_package}}.qgis_plugin_tools.tools.custom_logging import setup_logger, teardown_logger
```
Comment (LKajan) [inline {{cookiecutter.project_directory}}/{{cookiecutter.plugin_package}}/plugin.py:8]: I see what you are trying to do here. If the package name is very long we'll face the same issue also elsewhere e.g. next import line.
So maybe we should just accept these problems with generated package. What do you think?

**Review by LKajan — APPROVED** [review]

File: {{cookiecutter.project_directory}}/{{cookiecutter.plugin_package}}/plugin.py (hunk 2 above)
Comment (LKajan) [reply {{cookiecutter.project_directory}}/{{cookiecutter.plugin_package}}/plugin.py:8]: I approved this like that but we should handle these somehow uniform. Could you add a issue about this.

File: {{cookiecutter.project_directory}}/test/test_plugin.py (hunk 1 above)
Comment (Joonalai) [reply {{cookiecutter.project_directory}}/test/test_plugin.py:5]: [Here](https://github.com/GispoCoding/qgis_plugin_tools/blob/master/tools/resources.py#L79) is the code, it just replaces the whitespace. That is used in QGIS setting keys for example, so there should be no whitespace.

File: {{cookiecutter.project_directory}}/{{cookiecutter.plugin_package}}/plugin.py (hunk 2 above)
Comment (Joonalai) [reply {{cookiecutter.project_directory}}/{{cookiecutter.plugin_package}}/plugin.py:8]: Yes, we should just accept these problems. I will merge this and add issue to fix CI tests.

**Pull request merged**